src/main/resources/elasticsearch/addOnInfoAndVersions-mappings.json

```diff
@@ -14,7 +14,8 @@
         "type": "text"
       },
       "tags": {
-        "type": "keyword"
+        "type": "keyword",
+	"index": "not_analyzed"
```

Comment:
Yes , this is very important . I did read what you pointed out but for some reason if I remember right then without explicitly denoting it as non analysed , it was still getting tokenized and hence not matching perfectly. Actually the - in "form-entry" acts as a delimiter and it gets tokenized. I'll try changing it and get back.